### PR TITLE
Update test with Python 3.10

### PR DESCRIPTION
**Pull request by lu-wang-dl**

### CODE REVIEW

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8]
+        python-version: ["3.9"]
```
Comment (panchalhp-db) [inline .github/workflows/publish-to-pypi.yml:15]: Should this be 3.10?

File: .github/workflows/runtime-test.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: ["3.9", "3.10"]
```
Comment (panchalhp-db) [inline .github/workflows/runtime-test.yml:15]: Do we not have any supported versions that are still using 3.8?

Comment (lu-wang-dl) [reply .github/workflows/runtime-test.yml:15]: We have 10.4LTS using 3.8. But even we need to update the code for 10.4, we will merge into branch-0.2.6. It should not affect the main branch, right?

Comment (panchalhp-db) [reply .github/workflows/runtime-test.yml:15]: Ah right, that's correct.

**Review by panchalhp-db — APPROVED** [review]

**Pull request merged**